src/main/kotlin/inc/evil/d17map/dtos/FloorResponse.kt

```diff
@@ -1,9 +1,9 @@
 package inc.evil.d17map.dtos
 
-import inc.evil.d17map.entities.Building
+import java.util.UUID
 
 class FloorResponse (
-    val id: String,
+    val id: UUID,
```

Comment:
Tutaj widzę pole `name` które w Request jest jako `floorName`, to tu też bym tak dał